src/js/components/MosaicView.js

```diff
@@ -204,3 +268,43 @@ ChannelMetadata.propTypes = {
 
 const buildMediaURI = (locationName, cameraName, channelName, filename) =>
   `${locationName}/${cameraName}/${channelName}/${filename}`
+
+function frameStep(vidID, timeDelta) {
+  console.log('frame delta is: ',timeDelta)
+  const video = _getById(vidID)
+  pauseVideo(video)
+  if (timeDelta < 0 && video.currentTime < 0) {
+    video.currentTime = 0
+  } else if (timeDelta > 0 && video.currentTime > video.duration) {
+    video.currentTime = video.duration
+  } else {
+    video.currentTime = video.currentTime + timeDelta
+  }
+}
+
+window.onkeydown = videoControl
+
+function videoControl(e) {
+  const video = document.querySelector(".view-video.has-focus video")
+  if (!video) {
+    return
+  }
+  const key = e.code
+  let timeDelta = 0
+  switch(key) {
+    case "ArrowLeft":
+      timeDelta = BACK
+      break
+    case "ArrowRight":
+      timeDelta = FORWARD
```

Comment:
Don't you need a `break` statement here too?

Reply:
No, there's only a need to stop executing the case to prevent `timeDelta` being set twice.